tests/subsys/net/lib/nrf_cloud/prj.conf

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+# ZTest
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+
+# Dependencies
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_FPU=y
+CONFIG_CJSON_LIB=y
+
+# Mocking
+CONFIG_UNITY=y
```

Comment:
I guess you should choose either ZTest or Unity. With Unity you get automatically CMock which provides the mocking functionality.
You can discuss this within your team and if you need examples, look at SMS library tests at tests/lib/sms.

Reply:
I changed to Unity only, thanks